zipline/data/benchmarks.py

```diff
@@ -24,5 +25,14 @@ def get_benchmark_returns(symbol, start_date, end_date):
     start_date is **not** included because we need the close from day N - 1 to
     compute the returns for day N.
     """
-    df = web.DataReader(symbol, 'google', start_date, end_date)
-    return df["Close"].sort_index().tz_localize('UTC').pct_change(1).iloc[1:]
+    df = web.DataReader(symbol, 'google', start_date, end_date) 
+    df.index = df.index.tz_localize('UTC')
+
+    calendar = get_calendar("NYSE")
+    start_index = calendar.all_sessions.searchsorted(start_date)
```

Comment:
As I mentioned in the commit message Google Finance price data for SPY missing some dates, for example 2012-02-02, which causes algorithms to break with
    KeyError: 'the label [2012-02-02 00:00:00+00:00] is not in the [index]'